docs/how-it-works.md

```diff
@@ -22,23 +22,28 @@ The `.api/` directory is where the CLI installation process stores all of its in
 
 ```
 .api/
-├── api.json             // The `package-lock.json` equivalent that records
-|                        // everything that's installed, when it was installed,
-|                        // what the original source was, and what version of
-|                        // `api` was used.
+├── api.json               // The `package-lock.json` equivalent that records
+|                          // everything that's installed, when it was installed,
+|                          // what language was generated, what the original
+|                          // source was, and what version of `api` was used.
 └── apis/
+    ├── hoot/
     ├── readme/
-    |   ├── node_modules/
-    │   ├── index.js
-    │   ├── index.d.ts   // All types for the SDK, ready to use in your IDE.
+    │   ├── dist/          // The compiled source for your SDK. For JS languages
+    |   |                  // we offer CJS and ESM compatible builds.
```

Comment:
i think saying we have multiple builds might imply that they're separate SDKs? what do you think of this instead?

```suggestion
    │   ├── dist/          // The compiled source for your SDK. For JS,
    |   |                  // we offer a single SDK that's compatible with both ESM and CJS.
```